core/templates/components/common-layout-directives/common-elements/sharing-links.component.html

```diff
@@ -5,7 +5,7 @@
       <span class="oppia-icon-accessibility-label">Facebook</span>
     </a>
   </div>
-  <div class="fx-main-center fx-cross-center padding-0-2">
+  <div class="fx-main-center fx-cross-center padding-0-2 twit">
```

Comment:
Hm, `twit` seems too non-descriptive. Can we call it `twitter-icon` or similar?

Reply:
sure